internal/parse/parser.go

```diff
@@ -300,20 +300,22 @@ func (p *Parser) parseInputExpression() (*InputPart, bool, error) {
 	return nil, false, nil
 }
 
-// parseStringLiteral parses quoted expressions and ignores their content.
+// parseStringLiteral parses quoted expressions and ignores their content
+// including escaped quotes.
```

Comment:
Maybe add this:
```
+// Returns a pointer to a BypassPart, true and nil on success.
+// Returns nil, false and an error if an error was found.
+// Returns nil, false and nil if no string literal was found.
```

Reply:
I think for changing the comments to this form we could do it all at once in a separate PR so that everything remains  consistent.